JS Basics/Strings/Problem_2/index.js

```diff
@@ -0,0 +1,19 @@
+function correctBrackets(str) {
+    let bracketsArr = [];
+
+    for(let i = 0; i < str.length; i++) {
+        if(str[i] == '(') {
+            bracketsArr.push(str[i]);
+        } else if(str[i] == ')') {
+            if(bracketsArr.length == 0) {
+                return "Incorrect";
+            }
+            bracketsArr.pop();
+        }
+    }
+    return bracketsArr.length == 0 ? "Correct" : "Incorrect";
+}
+
+const strArr = [ ')(a+b))'];
+const sample = strArr.toString();
+correctBrackets(sample);
```

Comment:
The function input should be the first element of the array.
Using arr.toString will combine all elements in a single string.